app/routes/authenticated.js

```diff
@@ -0,0 +1,10 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class AuthenticatedRoute extends Route {
+  @service session;
+
+  beforeModel(transition) {
+    this.session.requireAuthentication(transition, 'login');
+  }
+}
```

Comment:
Where do you use this route?

Reply:
Not using this route but trying to resolve error (not redirect to points after login `The route index was not found`)

Reply:
Actually, as mentioned in the `ember-simple-auth` documentation, we can extend all the routes that needs authentication from this route.

Reply:
I have tried using nested authenticated routes as mentioned in the addon docs, it's not working (not loding points index view), and checked on example app too, they are not using this approch in it.

I have use it like this:
`
this.route('authenticated', { path: '' }, () => {
  this.route('points', { path: '' }, () => {
      this.route('index', { path: '/' });
    });
});
`